src/hooks/useTheme.test.tsx

```diff
@@ -6,11 +6,16 @@ import {
 } from '../components/BrandConfigProvider';
 import * as baseTheme from '../components/BrandConfigProvider/theme/base';
 import { useTheme } from './useTheme';
+import { NavigationProvider } from '../common/NavigationProvider';
+
+jest.unmock('@react-navigation/native');
```

Comment:
Not realizing that this was mocked globally under `__mocks__` made for a fun afternoon of  debugging react navigation source code, yarn removing, reinstalling, blowing away node_modules several times.....   All I needed was this line. 😁

Reply:
> Not realizing that this was mocked globally under `__mocks__` made for a fun afternoon of debugging react navigation source code, yarn removing, reinstalling, blowing away node_modules several times..... All I needed was this line. 😁

Oh no! 😭